data/ggc.txt

```diff
@@ -60,10 +60,14 @@ ixsforall-mnl1 pmn4vg5aa-hoae
 lvlt-arn1 a8au-5goe
 lvlt-bna1 a8au-co5e
 lvlt-bos1 a8au-cvne
+lvlt-dus1 a8au-q0ne
 lvlt-ewr1 a8au-xfge
 lvlt-iad3 a8au-p5qs
 lvlt-iad4 a8au-p5qz
 lvlt-lax1 a8au-a5me
+lvlt-lhr2 a8au-aigl
```

Comment:
I don't see these new ggc servers added in the hosts file, are they available?

Reply:
Sorry for my mistake

Reply:
Line 4885